### PR TITLE
Create lesson

### PR DESCRIPTION
**Pull request by YutakaYamasaki**

### CODE REVIEW

File: db/migrate/20180904094955_create_words.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class CreateWords < ActiveRecord::Migration[5.2]
   def change
     create_table :words do |t|
-      t.string :category_id
+      t.string :category_id, foreign_key: true
```
Comment (inahsanjuan) [inline db/migrate/20180904094955_create_words.rb]: @YutakaYamasaki `category_id` should be `integer`.
Or you can use `t.references :category`

File: db/migrate/20180904100034_create_word_answers.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class CreateWordAnswers < ActiveRecord::Migration[5.2]
   def change
     create_table :word_answers do |t|
-      t.string :word_id
+      t.string :word_id,foreign_key: true
```
Comment (inahsanjuan) [inline db/migrate/20180904100034_create_word_answers.rb]: @YutakaYamasaki `t.references :word`

File: db/migrate/20180905130120_create_lessons.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLessons < ActiveRecord::Migration[5.2]
+  def change
+    create_table :lessons do |t|
+      t.string :user_id, foreign_key: true
+      t.string :category_id, foreign_key: true
```
Comment (inahsanjuan) [inline db/migrate/20180905130120_create_lessons.rb]: @YutakaYamasaki Use `references` here

File: db/migrate/20180906182028_create_lesson_words.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateLessonWords < ActiveRecord::Migration[5.2]
+  def change
+    create_table :lesson_words do |t|
+      t.integer :lesson_id, foreign_key: true
+      t.integer :word_id, foreign_key: true
+      t.integer :word_answer_id, foreign_key: true
```
Comment (inahsanjuan) [inline db/migrate/20180906182028_create_lesson_words.rb]: @YutakaYamasaki Use `references` here

File: app/controllers/lessons_controller.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+class LessonsController < ApplicationController
+	def create
+		@user = current_user
```
Comment (inahsanjuan) [inline app/controllers/lessons_controller.rb]: @YutakaYamasaki I want to know why you assigned `current_user` to `@user`, when you can access `current_user` in your views.

**Review by inahsanjuan — CHANGES_REQUESTED** [review]

File: app/controllers/lessons_controller.rb (hunk 5 above)
Comment (YutakaYamasaki) [reply app/controllers/lessons_controller.rb]: I was going to understand by making @user = current_user.
However, I understood it so I will fix it.

**Review by inahsanjuan — APPROVED** [review]

**Pull request merged**